pontoon/teams/views.py

```diff
@@ -129,12 +129,17 @@ def ajax_permissions(request, locale):
             }
         )
 
-    managers = l.managers_group.user_set.all()
-    translators = l.translators_group.user_set.exclude(pk__in=managers).all()
-    all_users = User.objects.exclude(pk__in=managers).exclude(pk__in=translators).exclude(email='')
+    managers = l.managers_group.user_set.order_by('email')
+    translators = l.translators_group.user_set.exclude(pk__in=managers).order_by('email')
+    all_users = (
+        User.objects
+            .exclude(pk__in=managers | translators)
+            .exclude(email='')
+            .order_by('email')
+    )
 
     contributors_qs = User.translators.with_translation_counts(None, Q(locale=l), None)
-    contributors = set(contributor.email for contributor in contributors_qs)
+    contributors = {contributor.email for contributor in contributors_qs}
```

Comment:
Isn't that the same as before?

Reply:
also - every time i read this code i get confused because the var has `_qs` and has a `Q` in the args that generate it - but its not really a `_qs` - can we rename it ?

Reply:
Actually, Personally I would remove this temporary variable and do a simple inline function call.

Reply:
@mathjazz okay... I'm a syntax sugar addict :D I'll revert that.